src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationCodeGenerator.vb

```diff
@@ -227,25 +219,7 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
                 Else
                     Debug.Fail("Unexpected MyApplication.ShutdownMode")
                 End If
-
-                '    Me.HighDpiMode = HighDpiMode.xxx
-                Dim HighDpiValue As String
-                Select Case MyApplication.HighDpiMode
-                    Case 0
-                        HighDpiValue = HighDpiMode_DpiUnaware
-                    Case 1
-                        HighDpiValue = HighDpiMode_SystemAware
-                    Case 2
-                        HighDpiValue = HighDpiMode_PerMonitor
-                    Case 3
-                        HighDpiValue = HighDpiMode_PerMonitorV2
-                    Case 4
-                        HighDpiValue = HighDpiMode_DpiUnawareGdiScaled
-                    Case Else
-                        HighDpiValue = String.Empty
-                End Select
-                AddFieldAssignment(Constructor, HighDpiModeFieldName, HighDpiModeFieldName, HighDpiValue)
```

Comment:
Removing the code generation doesn't sound like a proper fix: we introduced it for .NET Core by surfacing it in the project properties UI; removing this would mean that changes in the UI would not be reflected in the actual project.

Could you instead check what target framework is using to decide if we should show the property or generate its code? 

cc: @KlausLoeffelmann

Reply:
I wasn't aware that we are targeting .NET Framework with the new templates?
We won't be running into an issue with .NET Core 3.1, since it was not supported for the VB Application Framework to begin with.
For .NET 5, yes, we might be running into problems. But also, .NET 5 is out of support, and since we only had a very rudimentary AppFramework support solution via templates for it at the time to begin with, I am not sure, if this is something, which _can_ be properly addressed.

Reply:
Apologies @melytc, should've checked on this property before removing it. @melytc and @KlausLoeffelmann, I've updated to only write to the myapp file in netcore projects, and now the property only appears in netcore 6+ projects

Reply:
Can you confirm this is now the correct behavior?

Reply:
This should be the right thing to do.
We introduced the property in .NET 6.